File: kombarika/com/district/test/controller/CommuneController.java
package com.district.test.controller;


import com.district.test.repository.CommuneRepository;
import com.district.test.entity.Commune;
import org.springframework.http.*;
import org.springframework.beans.factory.annotation.Autowired;
import org.springframework.web.bind.annotation.*;


@RestController
@CrossOrigin("*")
@RequestMapping(path = "commune")
public class CommuneController
 {

	@Autowired
	private CommuneRepository repository;


	@PostMapping()
	public ResponseEntity<Commune> save(@RequestBody Commune commune){
	 	return ResponseEntity.ok(repository.save(commune));
	}
	@PutMapping()
	public ResponseEntity<Commune> update(@RequestBody Commune commune){
	 	return ResponseEntity.ok(repository.save(commune));
	}
	@DeleteMapping()
	public void delete(@RequestBody Commune commune){
	 	repository.delete(commune);
	}
	@GetMapping()
	public ResponseEntity<Iterable<Commune>> findAll(){
	 	return ResponseEntity.ok(repository.findAll());
	}





}

File: kombarika/com/district/test/controller/CommuneController.cs
namespace com.district.test.controller;


using System.Diagnostics;
using Microsoft.AspNetCore.Mvc;
using com.district.test.repository.CommuneContext;
using com.district.test.entity;
using ;


[ApiController]
[Route("api/[[commune]]")]
public class CommuneController : Controller {

	private readonly CommuneDbContext _context;
	private readonly ILogger<CommuneController> _logger;


	[HttpPost]
	public ActionResult<Commune> save([FromBody] Commune commune){
	 	_context.Commune.Add(commune);
		_context.SaveChanges();
		return Ok();
	}
	[HttpPut]
	public ActionResult<Commune> update([FromBody] Commune commune){
	 	var temp = commune;
		_context.SaveChanges();
		return Ok();
	}
	[HttpDelete]
	public ActionResult<Commune> delete([FromBody] Commune commune){
	 	_context.Commune.Remove(commune);
		_context.SaveChanges();
	}
	[HttpGet]
	public ActionResult<IEnumerable<Commune>> findAll(){
	 	return Ok(_context.Commune.ToList());
	}

	public CommuneController(CommuneDbContext context) { _context = context; }



}
